frontend/src/lib/k8s/event.ts

```diff
@@ -69,7 +71,7 @@ class Event extends makeKubeObject<KubeEvent>('Event') {
           return `${k}=${fieldSelector[k]}`;
         })
         .join(','),
-      limit: '500',
+      limit: EVENTS_LIMIT,
```

Comment:
I am not very confident with this change, as I am not sure how this can impact the details view. But it's probably fine.

Reply:
I think the previous limit didn't had any effect as we were not using in the api request we were making